src/main/java/overlap_crap_fix_locations_later/application_business/CalculateSectionHoursInteractor.java

```diff
@@ -0,0 +1,26 @@
+package overlap_crap_fix_locations_later.application_business;
+
+import retrieve_timetable_use_case.application_business.BlockModel;
+import retrieve_timetable_use_case.application_business.SectionModel;
+
+
+/**
+ * A helper interactor responsible for calculating the # of hours in a section.
+ * FIXME: Not sure if this deserves its own interactor, but I feel like if we ever changed how Section time is
+ *  calculated (idk, maybe we start incorporating UofT time into the formal start), that might be a reason below... that is not
```

Comment:
Note: Might be a good idea to remove the FIXME. I'm pretty sure having its own interactor is justifiable enough here. 

This is straight up something I should do but I'll still comment on it for when we merge to main.